build_product_images.py

```diff
@@ -10,13 +10,19 @@
     build_product_images.py --product zookeeper,kafka --image_version 0.1.0 --push
```

Comment:
This example was already out-of-date before this PR (as product-version is omitted): we could either drop it (as the information is there with the -h option) or add a couple of examples to reflect the newest changes.

Reply:
Updated.